### PR TITLE
[FEATURE] Add checker to ensure test files have fixture definition

### PR DESCRIPTION
**Pull request by TrangPham**
Checks that existing test files have IntegrationTestFixture defined that is not commented out.
Assumes that a test file contains snippet definitions

I have spot checked that commented out IntegrationTestFixture would show as a violation here.

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Comment by TrangPham** [comment]
Was able to update the pipeline to use PythonScript task defined by azure
Documentation: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/python-script-v0?view=azure-pipelines&viewFallbackFrom=azure-devops

Output: https://dev.azure.com/great-expectations/great_expectations/_build/results?buildId=316822&view=logs&j=5830d702-cdd6-534b-05a8-6d067116ce45&t=dc1bd7b1-7baf-5a26-6750-ad2718c1c684

File: ci/checks/check_integration_test_gets_run.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+"""
+
+import pathlib
+import shutil
+import subprocess
+import sys
+from typing import Set
+
+IGNORED_VIOLATIONS = [
+    # TODO: Add IntegrationTestFixture for these tests or remove them if no longer needed
```
Comment (Shinnnyshinshin) [inline ci/checks/check_integration_test_gets_run.py:28]: ❤️

**Review by Shinnnyshinshin — APPROVED** [review]
This is great @TrangPham. Thank you!

File: ci/checks/check_integration_test_gets_run.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+def main() -> None:
+    check_dependencies("grep", "sed", "sort", "xargs")
+    project_root = pathlib.Path(__file__).parent.parent.parent
+    path_to_tests = project_root / "tests"
+    assert path_to_tests.exists()
+
+    new_violations = (
+        get_test_files(path_to_tests)
+        .difference(get_test_files_in_test_suite(path_to_tests))
+        .difference(IGNORED_VIOLATIONS)
```
Comment (anthonyburdi) [inline ci/checks/check_integration_test_gets_run.py:168]: ❤️

**Review by anthonyburdi — APPROVED** [review]
🙇 Thank you!

File: ci/azure-pipelines-dev.yml [hunk 3]
```diff
@@ -226,7 +236,7 @@ stages:
             scriptPath: 'ci/checks/check_name_tag_snippets_referenced.py'
             failOnStderr: true
           name: NameTagSnippetReferencedChecker
-
+
```
Comment (TrangPham) [inline ci/azure-pipelines-dev.yml]:
```suggestion

```

**Pull request merged**